backend/src/main/java/gov/cdc/usds/simplereport/api/devicetype/DeviceTypeMutationResolver.java

```diff
@@ -5,25 +5,26 @@
 import gov.cdc.usds.simplereport.api.model.errors.IllegalGraphqlArgumentException;
 import gov.cdc.usds.simplereport.db.model.DeviceType;
 import gov.cdc.usds.simplereport.service.DeviceTypeService;
-import graphql.kickstart.tools.GraphQLMutationResolver;
-import org.springframework.stereotype.Component;
+import lombok.RequiredArgsConstructor;
+import org.springframework.graphql.data.method.annotation.Argument;
+import org.springframework.graphql.data.method.annotation.MutationMapping;
+import org.springframework.stereotype.Controller;
 
-@Component
-public class DeviceTypeMutationResolver implements GraphQLMutationResolver {
+@Controller
+@RequiredArgsConstructor
+public class DeviceTypeMutationResolver {
 
-  private final DeviceTypeService _dts;
+  private final DeviceTypeService deviceTypeService;
 
-  public DeviceTypeMutationResolver(DeviceTypeService dts) {
-    _dts = dts;
-  }
-
-  public DeviceType createDeviceType(CreateDeviceType input)
+  @MutationMapping
```

Comment:
if you mean can we combine the the query and mutation resolvers, yes

- I just kept the changes to minimal size in this PR
- I am not sure that would be a good thing or a bad thing

Reply:
That's exactly was what I was suggesting. To just combine the query and mutation files together into one file. The more I think about it, I think we probably shouldn't do it. It will probably just end up making things more cluttered and confusing.